javascript/javascript1/week1/startup_name.js

```diff
@@ -0,0 +1,11 @@
+let firstWords =['Flex', 'Perfect', 'Extreme', 'Fort', 'Newly']
+let secondWords = ['Corporate', 'IPS', 'APS', 'Limited', 'Organization']
+let randomNumber = Math.floor(Math.random(0) * 10) + 0;
```

Comment:
```suggestion
let randomNumber = Math.random() *10
```
What do you think?

Reply:
I have tried this code but I don't know my startUpName variable become undefined

Reply:
Yes. I have forgot to add Math.floor
```suggestion
let randomNumber =   Math.floor(Math.random() *firstWords.length) 
```

Reply:
Thanks :) I will change it.